apps/console/src/features/applications/models/application-inbound.ts

```diff
@@ -72,6 +80,28 @@ export interface OAuth2PKCEConfigurationInterface {
     supportPlainTransformAlgorithm?: boolean;
 }
 
+export interface ClientAuthenticationConfigurationInterface {
```

Comment:
Let's add a small comment to describe what this interface is.
 Check other places.

Reply:
added with [a5f0e39](https://github.com/wso2/identity-apps/pull/4284/commits/a5f0e399c3a9fa90a48a937face8fe7d2f554faf)